make/buf/scripts/release.bash

```diff
@@ -100,15 +101,20 @@ for os in Darwin Linux Windows; do
       buf \
       protoc-gen-buf-breaking \
       protoc-gen-buf-lint; do
-      CGO_ENABLED=0 GOOS=$(goos "${os}") GOARCH=$(goarch "${arch}") \
-        go build -a -ldflags "-s -w" -trimpath -buildvcs=false -o "${dir}/bin/${binary}${extension}" "${DIR}/cmd/${binary}"
+      if [ "${arch}" == "armv7l" ]; then
+        CGO_ENABLED=0 GOOS=$(goos "${os}") GOARCH=$(goarch "${arch}") GOARM=7 \
+          go build -a -ldflags "-s -w" -trimpath -buildvcs=false -o "${dir}/bin/${binary}${extension}" "${DIR}/cmd/${binary}"
+      else
+        CGO_ENABLED=0 GOOS=$(goos "${os}") GOARCH=$(goarch "${arch}") \
+          go build -a -ldflags "-s -w" -trimpath -buildvcs=false -o "${dir}/bin/${binary}${extension}" "${DIR}/cmd/${binary}"
+      fi
       cp "${dir}/bin/${binary}${extension}" "${binary}-${os}-${arch}${extension}"
     done
   done
 done
 
 for os in Darwin Linux Windows; do
-  for arch in x86_64 arm64; do
+  for arch in x86_64 arm64 armv7l; do
```

Comment:
```suggestion
  for arch in x86_64 arm64 armv7; do
    if [ "${arch}" == "armv7" ] && [ "${os}" != "Linux" ]; then
      continue
    fi
```

Since we're only building for Linux, we'll need to add that condition here also.